src/services/EcoBridge/OmniBridge/OmniBridge.reducers.ts

```diff
@@ -0,0 +1,155 @@
+import { TransactionReceipt } from '@ethersproject/abstract-provider'
+
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { TokenList } from '@uniswap/token-lists'
+
+import { BridgeDetails, BridgingDetailsErrorMessage, OmniBridgeList, SyncState } from '../EcoBridge.types'
+import { omniTransactionsAdapter } from './OmniBridge.adapter'
+import { InitialState, OmnibridgeTransactionMessage, OmniBridgeTxn } from './OmniBridge.types'
+
+const initialState: InitialState = {
+  transactions: omniTransactionsAdapter.getInitialState({}),
+  lists: {},
+  listsStatus: SyncState.IDLE,
+  bridgingDetails: {},
+  bridgingDetailsStatus: SyncState.IDLE,
+  lastMetadataCt: 0,
```

Comment:
What's this variable?

Reply:
Counter of `getBridgingMetadata` calls. It's way of cancelling requests. Below we use it to check if current request is the last one 
```js
  if (requestId !== state.lastMetadataCt) { // do something }
```